src/DonorDashboards/Tabs/EditProfileTab/Tab.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Give\DonorDashboards\Tabs\EditProfileTab;
+
+use Give\DonorDashboards\Tabs\Contracts\Tab as TabAbstract;
+use Give\DonorDashboards\Tabs\EditProfileTab\ProfileRoute;
+use Give\DonorDashboards\Tabs\EditProfileTab\LocationRoute;
+
+/**
+ * @since 2.10.0
+ */
+class Tab extends TabAbstract {
```

Comment:
@henryholtgeerts What's the story with this introduction?

Reply:
Hmm...not entirely sure why it thinks that's new...it exists in the same way on `develop`. Not a change I made as part of this PR.

Reply:
Possibly auto-formatting by husky?